web/src/ee/features/evals/components/evaluator-form.tsx

```diff
@@ -457,7 +493,7 @@ export const InnerEvalConfigForm = (props: {
   }
 
   return (
-    <Form {...form}>
+    <Form {...form} key={props.existingEvaluator?.updatedAt?.toISOString()}>
```

Comment:
Mini-nit: why not just the evaluator id and the mode? this seems somewhat complex

Reply:
We would not force a remount if id and mode are staying the same on updates of the existingEvaluator. So I added the `updatedAt` timestamp to force remount on update. However, the form did not handle that well at all, so I added a `router.reload()` to force reload the page to show consistent state

Reply:
I'll remove that